### PR TITLE
Task.52213

### PR DESCRIPTION
**Pull request by thonght-3051**
## Related Tickets
- [#52213](https://edu-redmine.sun-asterisk.vn/issues/52213)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
ezgif com-gif-maker (5)

### CODE REVIEW

**Review by dangthibichngocb — APPROVED** [review]

**Comment by thonght-3051** [comment]
ready

File: app/views/auth/login.html.erb [hunk 1]
```diff
@@ -1,2 +1,26 @@
...
+          <%= f.password_field :password, placeholder: t("password") %>
+        </div>
+        <%= f.submit t("login") %>
+      <% end %>
+    </div>
+    <%= t "text_register" %>
+    <%= link_to t("register"), auth_register_path %>
+  </div>
+</div>
+</div>
```
Comment (ghost) [inline app/views/auth/login.html.erb]: thừa div này em

File: app/views/auth/login.html.erb [hunk 2]
```diff
@@ -1,2 +1,26 @@
...
+    <div class="title_container">
+      <h2><%= t "login" %></h2>
+    </div>
+    <div class="title_container">
+      <% flash.each do |message_type, message| %>
+        <div class="alert alert-<%= message_type %>"><%= message %></div>
+      <% end %>
+    </div>
+    <div class="row clearfix">
+      <%= form_for :session, url: auth_handle_login_path, html: { method: :post } do |f| %>
```
Comment (ghost) [inline app/views/auth/login.html.erb]: bỏ html: { method: :post } đc ko em

**Review by ghost — COMMENTED** [review]
cmt

File: app/views/auth/login.html.erb (hunk 2 above)
Comment (thonght-3051) [reply app/views/auth/login.html.erb]: đc nha a

**Comment by thonght-3051** [comment]
ready

File: app/controllers/auth_controller.rb [hunk 3]
```diff
@@ -15,6 +16,23 @@ def create
     end
   end
 
+  def login; end
+
+  def handle_login
```
Comment (huongnt-2545) [inline app/controllers/auth_controller.rb:21]: sao ko xử lý tại action `login` luôn em??

Comment (thonght-3051) [reply app/controllers/auth_controller.rb:21]: action đó em render ra view login thui ạ

**Comment by thonght-3051** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**